test/test_autograd.py

```diff
@@ -4980,6 +4980,17 @@ def assert_only_first_requires_grad(res):
                                          return_counts=return_counts)
                     assert_only_first_requires_grad(res)
 
+    def test_linalg_qr_r(self):
```

Comment:
This is a great test. Let's move it to test_linalg.py where the other qr tests live.

In the future I expect test_autograd.py to test autograd system-specific things.

Reply:
261133cdfc